gradle/profile_dev.gradle

```diff
@@ -21,22 +31,16 @@ bootRun {
 
 
 processResources {
-    filesMatching('**/logback-spring.xml') {
-        filter {
-            it.replace('#logback.loglevel#', logbackLoglevel)
-        }
```

Comment:
`filesMatching('**/logback-spring.xml') {`

copy this section to new version, also please check that log level is correct in dev and prod profiles (DEBUG in dev, INFO in prod)

Reply:
done